optimade/server/main.py

```diff
@@ -93,24 +85,16 @@ def load_entries(endpoint_name: str, endpoint_collection: MongoCollection):
     for name, collection in ENTRY_COLLECTIONS.items():
         load_entries(name, collection)
 
+# Add required OPTIMADE middleware
+for middleware in OPTIMADE_MIDDLEWARE:
+    app.add_middleware(middleware)
 
-# Add various middleware
+# Add additional CORS middleware
```

Comment:
This should come before the other middleware if I remember correctly.
One of our middleware (maybe the warnings one?) _needs_ to be last for it to properly work. Worrying that the tests didn't catch this.
The order of adding middleware matters.

Reply:
Huh, I guess the `CORSMiddlware` doesn't do enough to interfere with the warnings as it is just adding headers? Changing the order wasn't intention anyway, so I'll switch it back around